spec/factories/katalyst/content/items.rb

```diff
@@ -3,11 +3,15 @@
 FactoryBot.define do
   trait :content_item_defaults do
     heading { Faker::Lorem.word }
-    show_heading { true }
+    heading_style { "default" }
     background { Katalyst::Content.config.backgrounds.sample }
     depth { 0 }
   end
 
+  trait "header_hidden" do
```

Comment:
this seems unnecessary, it's easy to set this value with `key: :value` and the ergonomics of this trait aren't really helpful as they don't match the syntax (hidden vs none)